src/utils/resources/vm/utils/network/constants.ts

```diff
@@ -1,12 +1,22 @@
+/* eslint-disable require-jsdoc */
 import { V1Interface, V1Network } from '@kubevirt-ui/kubevirt-api/kubevirt';
 
 export type NetworkPresentation = {
   iface: V1Interface;
   network: V1Network;
 };
 
-export const interfacesTypes = {
+const typeHandler = {
+  get(target, prop) {
+    if (target[prop]) return target[prop];
+    return target.bridge;
+  },
```

Comment:
I think `return target[prop] ?? target.bridge` should work as well

Reply:
yes can work also